app/src/main/res/layout/item_horizontal_session.xml

```diff
@@ -90,6 +90,7 @@
                 android:layout_marginStart="12dp"
                 android:layout_marginTop="10dp"
                 android:lines="2"
+                android:lineSpacingExtra="@dimen/session_title_line_spacing"
```

Comment:
I think it's better to arrange this attribute as styles.xml, but your fix is enough for now!